test/unit/tracks/html-track-element-list.test.js

```diff
@@ -28,31 +28,47 @@ const genericHtmlTrackElements = [{
   track: track2
 }];
 
-q.module('HTML Track Element List');
+QUnit.module('HTML Track Element List');
 
-test('HTMLTrackElementList\'s length is set correctly', function() {
-  let htmlTrackElementList = new HTMLTrackElementList(genericHtmlTrackElements);
+QUnit.test('HTMLTrackElementList\'s length is set correctly', function() {
```

Comment:
if it isn't enabled yet, we should enable the eslint rule that allows us to wrap this in double quotes.